src/ImageSharp/Memory/Allocators/Internals/ManagedBufferBase.cs

```diff
@@ -43,5 +43,5 @@ public override void Unpin()
     /// Gets the object that should be pinned.
     /// </summary>
     /// <returns>The pinnable <see cref="object"/>.</returns>
-    protected abstract object GetPinnableObject();
+    protected abstract object? GetPinnableObject();
```

Comment:
This should rather throw when there is no pinnable object present.

Reply:
What exception do you want me to throw?

Reply:
Depends on the states when the underlying field is null. If it's because of disposal, it should be `ObjectDisposedException`.